layouts/partials/projects/get_project.html

```diff
@@ -0,0 +1,3 @@
+{{ $projectName := .projectName }}
+{{ $projectKey := printf "%s_%s_%d" $projectName site.Params.academicYear site.Params.semester }}
+{{ return $projectKey }}
```

Comment:
if this could return the reference to the exact data object, that would be great but i can't figure it out 😢

> probably got do with how partials are rendered as strings or something. hugo doesn't have a good way to create custom functions either